components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.ts

```diff
@@ -66,15 +83,20 @@ export class PolicyAddMembersComponent implements OnInit, OnDestroy {
 
   // Form info
   public expressionForm: FormGroup;
+  public expressionOutput: string;
+  public allIdentity: string;
+  public nameOrId: string;
+
 
   constructor(
     private store: Store<NgrxStateAtom>,
     private router: Router,
     fb: FormBuilder) {
 
     this.expressionForm = fb.group({
-      // Must stay in sync with error checks in policy-add-members.component.html
-      expression: ['', [Validators.required, Validators.pattern(Regex.patterns.NON_BLANK)]]
+      type: ['', [Validators.required, Validators.pattern(Regex.patterns.NON_BLANK)]],
```

Comment:
Since this is a dropdown and not a typeable field, do not need the pattern validator. (And, you are not even using it in the HTML--that's why you probably should not delete the comment on L76; it is there as a reminder for precisely that. 😁 )